flow/scripts/pdn.tcl

```diff
@@ -1,12 +1,8 @@
 source $::env(SCRIPTS_DIR)/load.tcl
 load_design 2_5_floorplan_tapcell.odb 1_synth.sdc "Starting PDN generation"
 
-if {[info exist ::env(PDN_TCL)]} {
-  source $::env(PDN_TCL)
-  pdngen -verbose
-} elseif {[info exist ::env(PDN_CFG)]} {
-  pdngen $::env(PDN_CFG) -verbose
```

Comment:
Maybe instead throw a warning about deprecation?

Reply:
We are going to remove it from OR - its already been deprecated for quite a while.

Reply:
I meant like 
```
elseif {[info exist ::env(PDN_CFG)]} {
  puts "[WARN][FLW-XXX] PDN_CFG variable is deprecated. Please use PDN_TCL"
```
but if it has been out of OR for a while then maybe it's not necessary. People tend to ignore warnings unless something is broken.

Reply:
It would have to be an error as deprecated implies it is still available.  They will have been getting deprecated warnings from pdngen itself already for quite a while.  All PDKs in OR have been converted.